### DISCUSSION TITLE
Проблемы с ютубом

### DISCUSSION THREAD

**Discussion by PerfectParadise** [Flowseal/zapret-discord-youtube#495] [General]
Доброго времени суток всем! Люди добрые помогите может быть кто-то сталкивался с такой проблемой!
сегодня после установки zapret 1.5.1 вроде ничего лишнего не тыкал сделал service_install.bat да и всё ютуб не трогал вроде!
И появилась такая проблема при просматривание видео или музыкальных клипов в ютубе в таком режиме 
![image](https://github.com/user-attachments/assets/1626e130-d651-4e67-8fcf-2b3e4d818892)
Видео сами по себе перелистываются бесконечно не пройдя и 2-3х секунд на видео и  сразу включается новое такое только в джемах происходит, если смотреть обычное видео просто то всё окей! 
![image](https://github.com/user-attachments/assets/6542ad8f-2cbb-4a87-a42f-b6d13d559970)
И Самое интересно что когда джем сворачиваешь в мини проигрыватель как на фото 
![image](https://github.com/user-attachments/assets/4932d97e-83af-4a64-88d2-85c3842239b7)
то всё работает как и должно быть.
В чем может быть проблема?? Помогите пожалуйста!

**Comment by scndjke** [comment]
Привет, кэш чистил ?) я браузер не закрывал пару месяцев =) тоже самое было)

**Reply by PerfectParadise** [reply]
Ага чистил

**Reply by scndjke** [reply]
Так, у меня тоже самое было)) почистил кэш, не помогло, просто новое окно открыл)) яндекс ?
Я просто, пофиксил и забыл, это опять палки в колеса что ли от ркн)

**Reply by PerfectParadise** [reply]
Гугл, не мне кажется я что-то все таки тыкнул или подхватил

**Reply by PerfectParadise** [reply]
фантастика, сегодня включил комп всё заработало  как и должно быть)))
